doc/usermanual/overview/overview.xml

```diff
@@ -103,16 +103,16 @@
     </para></listitem>
 
     <listitem><para>
-      Many darktable modules can be combined with blending operators for even more development
-      options.
+      A powerful mask feature gives you fine control over extent the module affects complete image
+      or its specific parts. To select image areas you draw a mask using various shapes and/or 
```

Comment:
Matter of taste, I think: I'd tend to say "You can select image areas by drawing a mask using various shapes, or by defining a parametric mask based on pixel values."

Reply:
merged